specification/implementation/structure/index.md

```diff
@@ -2,13 +2,12 @@
 title: CSIP structure
 ---
 # 4.	CSIP structure
-The preferred implementation of the conceptual model described in Principle 3.6 is a fixed physical (folder) structure which follows exactly the conceptual structure. While the CS IP doesn't prohibited alternative implementations of the conceptual model such implementations aren't recommended.
+The preferred implementation of the conceptual model described in Principle 3.6 http://earkcsip.dilcis.eu/specification/specification/principles/#principle-36 is a fixed physical (folder) structure which follows exactly the conceptual structure. While the CS IP doesn't prohibited alternative implementations of the conceptual model such implementations aren't recommended.
```

Comment:
Principle 3.6 http://earkcsip.dilcis.eu/specification/specification/principles/#principle-36
to
[Principle 3.6](http://earkcsip.dilcis.eu/specification/specification/principles/#principle-36)

Reply:
Fixed.